base/sparse/cholmod.jl

```diff
@@ -602,15 +602,15 @@ function copy{Tv<:VRealTypes}(A::Sparse{Tv}, stype::Integer, mode::Integer)
 end
 
 ### cholmod_check.h ###
-function print_sparse{Tv<:VTypes}(A::Sparse{Tv}, name::ASCIIString)
+function print_sparse{Tv<:VTypes}(A::Sparse{Tv}, name::String)
     cm = common()
     set_print_level(cm, 3)
     @isok ccall((@cholmod_name("print_sparse", SuiteSparse_long),:libcholmod), Cint,
```

Comment:
OK, so let's add an `isascii` call here, performance isn't critical in this function anyway.